.github/workflows/docker-build.yml

```diff
@@ -18,17 +18,18 @@ on:
         default: registry.scality.com
     secrets:
       REGISTRY_LOGIN:
-        required: true
+        required: false
```

Comment:
Why are you changing the default values in this file?

Reply:
Because we don't necessarily have the previous secrets, unless we send `github.repository_owner` or `secrets.GITHUB_TOKEN`. I wanted to mimic the behavior in [docker-build](https://github.com/scality/workflows/blob/main/.github/workflows/docker-build.yaml) from the `workflows` repo.

Reply:
If we supposedly do not have the previous secrets, please explain how other workflows in this repository are capable of using this workflow, without your changes?

Reply:
No but yes I know, I saw it more as a security that I wanted to add. In the sense that if you create a workflow later that uses this one and you forget to specify the secrets, the workflow will still work.
Anyways, I'll revert my changes as requested.

Reply:
It's a fair request, but:
* As we're not the maintainers of this repository, maybe the maintainers want precisely the opposite 🤷 
* This kind of change deserve its own pull request, not to be hidden behind the implementation of GHAS. Focus on your task, unless this code, without making any changes, is stopping you from completing it.